Client/Frontend/Widgets/SiteTableViewHeader.swift

```diff
@@ -67,6 +68,8 @@ class SiteTableViewHeader: UITableViewHeaderFooterView, ThemeApplicable, Reusabl
 
         showImage(model.isCollapsible)
         collapsibleState = model.collapsibleState
+
+        applyTheme(theme: theme)
```

Comment:
Theme is applied from the `applyTheme(theme:)` function already. This shouldn't be added, nor the theme = LightTheme.

Reply:
This should be fixed with https://github.com/mozilla-mobile/firefox-ios/pull/12249